### ISSUE TITLE
blendMode() and tint() with alpha do not work together

### ISSUE THREAD

**Issue by cdaein** [processing/processing#3032]
When blendMode is called and tint() with alpha value is used for images, the alpha value is ignored.

I was expecting something similar to how Photoshop works - the colors will blend AND the alpha value will be kept.

Please see example below.

```processing
PImage img1;
PImage img2;

float d = 0;

void setup() {
  size(400, 400);
  
  img1 = loadImage("1.jpg");
  img2 = loadImage("2.jpg");
}

void draw() {
  background(255);
  
  // comment next line to see the difference
  blendMode(MULTIPLY);
  
  tint(255);
  image(img1, 0, 0);
  
  tint(255, d);
  image(img2, 0, 0);
  
  d = map(mouseX, 0, width, 0, 255);
}
```

**Comment by cdaein** [comment]
forgot to add my system info - Processing 2.2.1 on Mac OS X 10.10.1 Yosemite

**Comment by benfry** [comment]
Please try with 3.0a5, which has several fixes for blending modes.

**Comment by JakubValtar** [comment]
I can confirm that this is fixed in 3.0a5.

**Comment by benfry** [comment]
Thanks for looking into it @JakubValtar
